lib/galaxy/schema/jobs.py

```diff
@@ -212,6 +212,12 @@ class JobOutput(Model):
     value: EncodedDataItemSourceId = Field(default=..., title="Dataset", description="The associated dataset.")
 
 
+class JobConsoleOutput(Model):
+    state: Optional[JobState] = Field(None, title="Job State", description="The current job's state")
```

Comment:
This is going to sounds weird but internally we track both a job_stderr/job_stdout and a tool_stderr/tool_stdout. Pulsar is worse at this than the other runners unfortunately. But I think the thing you're returning here is the tool_stdout - the output of the running tool. The job_stdout would be things like messages printed by SLURM and such. Can you update this description here to say Tool STDOUT and Tool STDERR. There are other places in comments where you refer to it as the job's stdout but I think it should be the job's tool stdout and such. It seems like there probably shouldn't be a distinction I imagine but we actually have disentangled a few problems this was and we shield the users from non-tool stuff they don't care about this way too.

Reply:
Gotcha.